src/app/qgscrashhandler.h

```diff
@@ -40,7 +40,7 @@ class APP_EXPORT QgsCrashHandler
      * Show the crash dialog.
      * @param stack The current stack of the crash point.
      */
-    static void showCrashDialog( const QList<QgsCrashReport::StackLine> &stack );
+    static void showCrashDialog( const QVector<QgsStackTrace::StackLine> &stack );
```

Comment:
Might be nice to typdef QVector<QgsStackTrace::StackLine> somewhere to avoid typing it all the time